.ruby-version

```diff
@@ -1 +1 @@
-2.1.3
+2.3.1
```

Comment:
The ruby version on master is tied to the version of Ruby which is already installed on the vagrant VM the developers use. Please don't change this file now.